pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+filterwarnings =
+    ignore::xbout.warning.xBOUTWarning
+    ignore:deallocating CachingFileManager.*, but file is not already closed. This may indicate a bug\.:RuntimeWarning
```

Comment:
We should actually fix whatever is causing this warning...

Reply:
I agree in principle, but this warning also seems to be thrown in `xarray`'s test suite (https://github.com/pydata/xarray/issues/3266), so I guess it's an `xarray` issue. The warnings seem to be thrown during garbage-collection (I think I saw a comment somewhere that it might be related to some sort of reference-cycle, but I forget where now), and I didn't want to go chasing into the guts of `xarray`.